src/main/java/com/vmware/vipclient/i18n/messages/service/CacheService.java

```diff
@@ -104,18 +104,16 @@ public void addCacheOfStatus(Map<String, String> dataMap) {
 
     public List<Locale> getSupportedLocalesFromCache() {
         List<Locale> locales = new ArrayList<Locale>();
-        Cache c = VIPCfg.getInstance().getCacheManager().getCache(VIPCfg.CACHE_L3);
+        Cache c = VIPCfg.getInstance().getCacheManager().getCache(VIPCfg.CACHE_L2);
```

Comment:
This is actually a bug that I just fixed. I found it by writing LocaleTest.testGetSupportedLocalesOfflineBundles().

Here is the bug description:
In LocaleService.getDisplayNamesFromCLDR, the list of supported locales is cached in CACHE_L2. But CacheService. getSupportedLocalesFromCache never find it in the cache because it has been looking for it in CACHE_L3. So it always tries to get it from the service every time.